src/Plugin/Helper/Data.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tweakwise\TweakwiseJs\Plugin\Helper;
+
+use Magento\Search\Helper\Data as Subject;
+use Tweakwise\TweakwiseJs\Model\Config;
+use Tweakwise\TweakwiseJs\Model\Enum\SearchType;
+
+class Data
+{
+    /**
+     * @param Config $config
+     */
+    public function __construct(
+        private readonly Config $config
+    ) {
+    }
+
+    /**
+     * If search type is not the default Magento search -> disable the Magento suggestions by setting empty URL
+     * @param Subject $subject
+     * @param string $result
+     * @return string
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     */
+    public function afterGetSuggestUrl(Subject $subject, string $result): string
+    {
+        if (!$this->config->isEnabled() || $this->config->getSearchType() === SearchType::MAGENTO_DEFAULT->value) {
```

Comment:
Kunnen we geen nettere manier bedenken om de `suggest` uit te schakelen?
De tweakwise module kan in principe ook uitgebreid worden voor een netter extension point om dit te bepalen.
In principe zou het vrijwel niet nodig hoeven te zijn om plugins te maken op eigen tweakwise code.
Dit voelt wel een beetje fragiel en flimsy.